templates/zerver/help/add-a-bot-or-integration.md

```diff
@@ -0,0 +1,51 @@
+# Add a Bot or Integration
+
+Bots and integrations are features that are accesible to all members of an organization.
+
+## To Add a Bot
+
+Bots can be created from the settings page:
+
+1. Click the cog (![cog](/static/images/help/cog.png)) in the upper right corner
```

Comment:
The other sections, such as **Organization Settings**, **Users & Bots**, etc. count too.